lib/_tls_wrap.js

```diff
@@ -918,7 +918,7 @@ Server.prototype.addContext = function(servername, context) {
 
   var re = new RegExp('^' +
                       servername.replace(/([\.^$+?\-\\[\]{}])/g, '\\$1')
-                                .replace(/\*/g, '[^\.]*') +
+                                .replace(/\*/g, '[^.]*') +
```

Comment:
There is no actual difference between any of the three variants there, so we could just keep it as `[^.]*`.